### DISCUSSION TITLE
Will the T3EX throttle be compatable with the new DCC-EX CSB1 command station

### DISCUSSION THREAD

**Discussion by brianjackson1** [DCC-EX/EX-T3-WiFi#124] [General]
I am looking at buying the new DCC EX CSB1 command station.  I have already built two throttles and was wondering whether the two would be compatable?  It may be a way around the lag when using two throttles.

**Comment by alex-code** [comment]
It should all be compatible.
I can't comment on the multi throttle lag without testing myself but I'm hoping to get a CSB1 in the new year.
